src/packages/utils.ts

```diff
@@ -74,19 +75,31 @@ export const exportToBlob = (
   });
 };
 
-export const exportToSvg = ({
+export const exportToSvg = async ({
   elements,
   appState = getDefaultAppState(),
   exportPadding,
-  metadata,
+  embedScene,
 }: Omit<ExportOpts, "getDimensions"> & {
   exportPadding?: number;
-  metadata?: string;
-}): SVGSVGElement => {
+  embedScene?: boolean;
+}): Promise<SVGSVGElement> => {
   const { elements: restoredElements, appState: restoredAppState } = restore(
     { elements, appState },
     null,
   );
+  let metadata = "";
+  if (embedScene) {
+    try {
+      metadata = await (
+        await import(/* webpackChunkName: "image" */ "../../src/data/image")
+      ).encodeSvgMetadata({
+        text: serializeAsJSON(restoredElements, restoredAppState),
+      });
+    } catch (err) {
+      console.error(err);
+    }
+  }
```

Comment:
Come think of it, shouldn't we put this into `scene/exports.ts:exportToSvg` so we don't duplicate the logic here?

Reply:
Sure we can, but right now it just expects metadata which is just appended to `svg` without any logic behind what the `metadata` is so either we remove the `metadata` altogether and replace it with `embedScene` or we leave it to be generic so tomorrow if we want to update the logic around metadata the changes will be in places where `scene/exports.ts:exportToSvg` it is used.

Reply:
Yeah, I thought since we're removing `metadata` from the public API we may as well remove it altogether and deduplicate the embedScene logic.